### PR TITLE
US32, US27

### PR DESCRIPTION
**Pull request by Yun-L**
- List multiple births
- display age with with individual

### CODE REVIEW

**Comment by Yun-L** [comment]
couldn't write any test cases for US27 because it's literally just printing a person's age with their name.

**Review by DQSevilla — APPROVED** [review]

File: verifier.py [hunk 1]
```diff
@@ -330,6 +347,8 @@ def verify():
             print(f"US34-ERR: Family {id} has couples who are large age differences")
 
     for id, individual in individualsDict.items():
+        # US27 Include person's current age when listing individuals
+        print(f"Individual: {id}, name: {individual['name']}, age: {individual['age']}")
```
Comment (DQSevilla) [inline verifier.py]: FYI: noah introduced a function `print_individual` in the utils, try using that

**Review by DQSevilla — CHANGES_REQUESTED** [review]
See comment

File: verifier.py (hunk 1 above)
Comment (Yun-L) [reply verifier.py]: got it, updated in 10722d7960084879b527035c7b1842cc22542f1b

**Review by DQSevilla — APPROVED** [review]

**Pull request merged**